Multisig/Data/Services/Safe Client Gateway Service/FiatCodesRequest.swift

```diff
@@ -9,19 +9,19 @@
 import Foundation
 
 struct FiatCodesRequest: JSONRequest {
-    var httpMethod: String {
-        "GET"
-    }
+    let chainId: Int
+    
+    var httpMethod: String { "GET" }
 
     var urlPath: String {
-        "/v1/balances/supported-fiat-codes"
+        "/\(chainId)/v1/balances/supported-fiat-codes"
```

Comment:
As I can see here https://github.com/gnosis/safe-client-gateway/blob/b152f3e3d27c85efeb96c9baf8b43c7f5f8a6d93/src/routes/balances.rs#L62 there is no need in chain id here